Source/WebKit/UIProcess/ios/forms/WKDatePickerPopoverController.mm

```diff
@@ -155,6 +157,28 @@ - (void)resetDatePicker
     [_delegate datePickerPopoverControllerDidReset:self];
 }
 
+- (void)assertAccessoryViewCanBeHitTested
+{
+    auto accessoryView = [_contentView accessoryView];
+    auto popoverView = self.viewIfLoaded;
+
+    RELEASE_ASSERT(accessoryView);
+    RELEASE_ASSERT(popoverView);
```

Comment:
Should we really be release asserting? I think it's better to have incorrect UI rather than induce a crash.

Reply:
Ah, this codepath is only exercised by layout/API tests. I think there's a slight advantage to release asserting in this case, since it means those layout tests will be useful in release configs as well.

Maybe it would help to make that clearer by adding `-ForTesting` suffixes to some of these methods?

Reply:
Oh! That's fine then, – `-ForTesting` would have helped 😅

Reply:
Thanks for the review!

I'll rename this new method (along with all callers) to add `ForTesting` to the end to clarify this.